Formula/julia.rb

```diff
@@ -2,54 +2,16 @@ class Julia < Formula
   desc "Fast, Dynamic Programming Language"
   homepage "https://julialang.org/"
   license all_of: ["MIT", "BSD-3-Clause", "Apache-2.0", "BSL-1.0"]
-  revision 2
   head "https://github.com/JuliaLang/julia.git", branch: "master"
 
   stable do
-    url "https://github.com/JuliaLang/julia/releases/download/v1.7.3/julia-1.7.3.tar.gz"
-    sha256 "06df2a81e6a18d0333ffa58d36f6eb84934c38984898f9e0c3072c8facaa7306"
+    url "https://github.com/JuliaLang/julia/archive/refs/tags/v1.8.0.tar.gz"
+    sha256 "fba888fdf86ab3e7c0a0d1247e78946fee1f7f5b4930588606a8ac6a13f92491"
```

Comment:
Any reason this is using GitHub generated tarball rather than official? 

https://github.com/JuliaLang/julia/releases/download/v1.8.0/julia-1.8.0.tar.gz

Reply:
Oops, picked the wrong link.